src/components/CustomSelectOption/CustomSelectOption.tsx

```diff
@@ -86,7 +86,7 @@ export const CustomSelectOption = ({
   );
 
   if (!!option && process.env.NODE_ENV === 'development') {
-    warn('Свойство option было добавлено по ошибке и будет удалено в 5.0.0.');
+    warn('Свойство option было добавлено по ошибке и будет удалено в v6.0.0.');
```

Comment:
```suggestion
    // TODO v6.0.0. Удалить св-во `option`
    warn('Свойство option было добавлено по ошибке и будет удалено в v6.0.0.');
```

есть вероятность пропустить `warn`, т.к. если будем искать депрекейтит через `TODO v6.0.0`

можно ещё у св-ва `option` поставить коммент `@deprecated` с комментом, что в 6.0.0 ему пязда будет